arch/x86/ectx.c

```diff
@@ -110,6 +110,30 @@ __sz ukarch_ectx_align(void)
 	return ectx_align;
 }
 
+void ukarch_ectx_sanitize(struct ukarch_ectx *state)
+{
+	UK_ASSERT(ectx_align); /* Do not call when not yet initialized */
+	UK_ASSERT(state);
+	UK_ASSERT(IS_ALIGNED((__uptr) state, ectx_align));
```

Comment:
Hmm, strange, I would have expected `checkpatch.pl` to issue a warning regarding whitespace between the cast and the variable itself, but the entire file seems to have it at every cast inside `IS_ALIGNED`.

Reply:
This whitespace in front of the casted variable is actually correct.